scripts/gha/it_workflow.py

```diff
@@ -60,13 +60,17 @@
 _LABEL_SUCCEED = "tests: succeeded"
 
 _COMMENT_TITLE_PROGESS = "### ⏳&nbsp; Integration test in progress...\n"
+_COMMENT_TITLE_PROGESS_FLAKY = "### Integration test with FLAKINESS (but still ⏳&nbsp; in progress)\n" 
 _COMMENT_TITLE_PROGESS_FAIL = "### ❌&nbsp; Integration test FAILED (but still ⏳&nbsp; in progress)\n" 
+_COMMENT_TITLE_FLAKY = "### Integration test with FLAKINESS (succeed after rerun the tests)\n"
```

Comment:
suggestion: instead of "succeed after rerun the tests" --> "succeeded after retry"